deploy.sh

```diff
@@ -0,0 +1,33 @@
+VAULT_TOKEN=$1
+ENVIRONMENT=$2
+
+# TODO: this needs to be derived from the env somehow; the only two vars this script can take are VAULT_TOKEN and ENVIRONMENT
+# environment could also be git branch; we don't have to explictly name the envs as they're going to be coming from branches
+GOOGLE_PROJECT=$3
+
+#jenkins needs access to the projects so it can deploy
+# command TBD (that's either already the case or we use some credential here to make it so)
+
+#render the endpoints json
+python lib/endpoints/endpointscfg.py get_openapi_spec main.BondApi --hostname $GOOGLE_PROJECT.appspot.com
+
+#deploy google endpoints
+gcloud -q endpoints services deploy linkv1openapi.json
```

Comment:
in order to make all gcloud commands work you'll have to authenticate with a service account, which you should grab from vault.  see how we do it during fiab populate, for example - https://github.com/broadinstitute/firecloud-develop/blob/dev/run-context/fiab/scripts/fiab-populate/sam_google_extensions_security.sh#L25

then you'll want to run `gcloud auth activate-service-account <path-to-svc-acct>`